rest/src/test/java/com/ericsson/bss/cassandra/ecchronos/rest/TestScheduleRepairManagementRESTImpl.java

```diff
@@ -64,6 +64,23 @@ public void setupMocks()
         ScheduleREST = new ScheduleRepairManagementRESTImpl(myRepairScheduler);
     }
 
+    @Test
```

Comment:
This test are actually testing nothing, you're mocking the return, which means that anything you pass to the function as a parameter will always return the t, and than checking if the same method are returning the mocked value